bazel/extensions/extensions_build_config.bzl

```diff
@@ -239,7 +247,8 @@ EXTENSIONS = {
 
     "envoy.resource_monitors.fixed_heap":               "//source/extensions/resource_monitors/fixed_heap:config",
     "envoy.resource_monitors.injected_resource":        "//source/extensions/resource_monitors/injected_resource:config",
-    "envoy.resource_monitors.downstream_connections":   "//source/extensions/resource_monitors/downstream_connections:config",
+    "envoy.resource_monitors.global_downstream_max_connections":   "//source/extensions/resource_monitors/downstream_connections:config",
```

Comment:
Just want to make sure this is not a cut-and-paste error but intentionally changing the name but not the value?

Reply:
I did a quick search and I think I answer my own question.

Reply:
Yep and we have a test that checks for these. So this would be an interesting change for us except we dont use it today in the control plane.